tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -24,6 +24,14 @@ describe('Metadata generation', () => {
     });
   });
 
+  describe('ValidExtensionControllerGenerator', () => {
```

Comment:
We already have specs for existing extensions for both v2 and v3 of OpenAPI.
Can you please extend these? Not throwing is too weak of an assertion.

Refs:
- https://github.com/lukeautry/tsoa/blob/master/tests/fixtures/controllers/methodController.ts#L158
- https://github.com/lukeautry/tsoa/blob/master/tests/unit/swagger/schemaDetails.spec.ts#L495
- https://github.com/lukeautry/tsoa/blob/master/tests/unit/swagger/schemaDetails3.spec.ts#L2144

Reply:
@WoH OK, I added the fixture/tests to methodController.ts, schemaDetails.spec.ts, schemaDetails3.spec.ts and metadata.spec.ts.